ArcGISRuntimeSDKQt_QMLSamples/Maps/ReadGeopackage/ReadGeoPackage.json

```diff
@@ -0,0 +1,5 @@
+{
+    "source" :  "ReadGeoPackage.qml",
+    "otherSourceFiles" :  [],
+    "name" : "Read GeoPackage"
```

Comment:
@acmorris This json file is missing the data information now (I thought it contained it before).

```
"dataItems" : [
        {
            "itemId" : "68ec42517cdd439e81b036210483e8e7",
            "path" : "~/ArcGIS/Runtime/Data/gpkg/AuroraCO.gpkg"
        }
    ]
```